http-rust-1.79-axum-scale-to-zero/README.md

```diff
@@ -0,0 +1,36 @@
+# Example for app scale-to-zero integration
```

Comment:
```diff
- # Example for app scale-to-zero integration
+ # Example Rust Axum application with internal scale-to-zero trigger
```
Basically just need to add Rust + Axum into the title.

Reply:
It is primarily an app scale-to-zero example, and the title just looks horribly long that way.